Resources/public/js/views/fields/ez-country-editview.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-country-editview', function (Y) {
+    "use strict";
+    /**
+     * Provides the field edit view for the country fields
+     *
+     * @module ez-country-editview
+     */
+
+    Y.namespace('eZ');
+
+    var FIELDTYPE_IDENTIFIER = 'ezcountry';
+
+    /**
+     * Country edit view
+     *
+     * @namespace eZ
+     * @class CountryEditView
+     * @constructor
+     * @extends eZ.SelectionEditView
+     */
+    Y.eZ.CountryEditView = Y.Base.create('countryEditView', Y.eZ.SelectionEditView, [], {
+
+        initializer: function () {
+            this.containerTemplate = '<div class="' + this._generateViewClassName(this._getName()) + ' ' + this._generateViewClassName(Y.eZ.SelectionEditView.NAME) +'"/>';
+        },
+
+        /**
+         * Returns the current country name.
```

Comment:
*the current country name* ? More *the country name corresponding to the given alpha2Code*